packages/db/src/schema/monitors/monitor.ts

```diff
@@ -36,6 +37,8 @@ export const monitor = sqliteTable("monitor", {
   name: text("name", { length: 256 }).default("").notNull(),
   description: text("description").default("").notNull(),
 
+  statusCode: text("status_code").default("200"),
```

Comment:
We should allow `null` and **not** default to `200` as we might break currently used monitors.